src/sqlodbc/opensearch_communication.cpp

```diff
@@ -432,14 +432,35 @@ OpenSearchCommunication::IssueRequest(
             credential_provider = Aws::MakeShared<
                 Aws::Auth::ProfileConfigFileAWSCredentialsProvider >(
                 ALLOCATION_TAG.c_str(), ESODBC_PROFILE_NAME.c_str());
+
+        LogMsg(OPENSEARCH_ALL,
+               (Aws::String("=== KEY = ")
+                + credential_provider->GetAWSCredentials().GetAWSAccessKeyId()
+                + " SECRET = "
+                + credential_provider->GetAWSCredentials().GetAWSSecretKey()
```

Comment:
Don't do this unless it's anonymized.  We do not want to put AWS credentials in the log.

Reply:
Sure, this was for debugging needs